lib/task-data/tasks/boot-livecd.js

```diff
@@ -0,0 +1,22 @@
+// Copyright 2015, EMC, Inc.
+
+'use strict';
+
+module.exports = {
+    friendlyName: 'Boot LiveCD',
+    injectableName: 'Task.Os.Boot.LiveCD',
+    implementsTask: 'Task.Base.Os.Install',
+    options: {
+        profile: 'boot-livecd.ipxe',
+        completionUri: 'renasar-ansible.pub',
```

Comment:
Do you have any script to help create the LiveCD so that ensure the completionUrl query is added into it? Or any document?

Previously, the completionUrl query is in our kickstart script, we are in the control of it. But if we define it in LiveCD, we pass the control to user. If user forgets this customization work, our workflow cannot detect this and the workflow will eventually hang until timeout.

Do you have every study whether we can always inject some command into the LiveCD booting, so we can take this to notify the boot LiveCD success.